R/load_deathrates.R

```diff
@@ -0,0 +1,63 @@
+#' Load deathrates
+#'
+#' Taking the input of a country or iso3c code, load the deathrates in
+#' the format required by [malariasimulation](https://github.com/mrc-ide/malariasimulation/).
+#'
+#' @param iso3c Country code to subset deathrates by.
+#' @param country Country name to subset deathrates by.
+#' @param year Year to subset deathrates by. Can be single value or vector.
+#' If format = 'asmr', year data is given in 5 year chunks. If format = 'nmr',
+#' year is available by single year. There is no difference in the distribution
+#' of values between the two formats.
+#' @param format Format to return deathrates by ('asmr' = by years divisible by
+#' 5, or 'nmr' = single year values). See 'peeps::merge_nmr_asmr' documentation
+#' for details.
+#'
+#' @return Data.frame of deathrates for country.
+#' @export
+load_deathrates <- function(iso3c = NULL, country = NULL, year = NULL,
```

Comment:
I think we need a check somewhere on the `iso3c` and `country` inputs. For example I can run:
`load_deathrates(iso3c = "AGO", country = "Benin")` with no error/warning. Probably good to write a test for this too.